Jenkins/Common/notify-github.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+curl -s "https://api.github.com/repos/studocu/studocu/statuses/$GIT_COMMIT" \
```

Comment:
```suggestion
curl -s "https://api.github.com/repos/studocu/design-figma-tokens/statuses/$GIT_COMMIT" \
```
Or am I misunderstanding this?

Reply:
Nope, you're absolutely right... Skipped that sorry... fixing.

Reply:
[bf0a2ee](https://github.com/StuDocu/design-figma-tokens/pull/19/commits/bf0a2eefd4ad60a8a01ba823aa49aca44e01e590)